lib/membrane/rtp/metrics.ex

```diff
@@ -3,34 +3,66 @@ defmodule Membrane.RTP.Metrics do
   Defines list of metrics, that can be aggregated based on events from membrane_rtp_plugin.
   """
 
+  alias Telemetry.Metrics
+
   @doc """
   Returns list of metrics, that can be aggregated based on events from membrane_rtp_plugin.
   """
-  @spec metrics() :: [Telemetry.Metrics.t()]
+  @spec metrics() :: [Metrics.t()]
   def metrics() do
     [
-      Telemetry.Metrics.counter(
+      Metrics.counter(
         "inbound-rtp.keyframe_request_sent",
         event_name: [Membrane.RTP, :rtcp, :fir, :sent]
       ),
-      Telemetry.Metrics.counter(
+      Metrics.counter(
         "inbound-rtp.packets",
         event_name: [Membrane.RTP, :packet, :arrival]
       ),
-      Telemetry.Metrics.sum(
+      Metrics.sum(
         "inbound-rtp.bytes_received",
         event_name: [Membrane.RTP, :packet, :arrival],
         measurement: :bytes
       ),
-      Telemetry.Metrics.last_value(
+      Metrics.last_value(
         "inbound-rtp.encoding",
         event_name: [Membrane.RTP, :inbound_track, :new],
         measurement: :encoding
       ),
-      Telemetry.Metrics.last_value(
+      Metrics.last_value(
         "inbound-rtp.ssrc",
         event_name: [Membrane.RTP, :inbound_track, :new],
         measurement: :ssrc
+      ),
+      Metrics.counter(
+        "inbound-rtp.nack",
```

Comment:
Just to be sure, I'd include `_sent` in the name to leave no questions
```suggestion
        "inbound-rtp.nack_sent",
```